iphone/Maps/UI/BottomMenu/TabBar/BottomTabBarViewController.swift

```diff
@@ -56,9 +56,14 @@ class BottomTabBarViewController: UIViewController {
   @IBAction func onSearchButtonPressed(_ sender: Any) {
     presenter.onSearchButtonPressed()
   }
-
+  
```

Comment:
Unnecessary spaces?

Reply:
I suppose it is Ok if I'll remove all Unnecessary spaces in the file?
<img width="509" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/6c6cda19-42e6-4a70-96d3-85d4e147f0cd">